src/main/java/org/sagebionetworks/bridge/services/AdherenceService.java

```diff
@@ -424,15 +424,16 @@ protected WeeklyAdherenceReport deriveWeeklyAdherenceFromStudyAdherenceReport(St
         weeklyReport.setTestAccount(report.isTestAccount());
         weeklyReport.setClientTimeZone(report.getClientTimeZone());
         weeklyReport.setCreatedOn(report.getCreatedOn());
-        weeklyReport.setNextActivity(report.getNextActivity());
-        StudyReportWeek week = report.getCurrentWeek();
-        if (week != null) {
-            weeklyReport.setSearchableLabels(week.getSearchableLabels());
-            weeklyReport.setRows(week.getRows());
-            weeklyReport.setByDayEntries(week.getByDayEntries());
-            weeklyReport.setWeeklyAdherencePercent(week.getAdherencePercent());
-            weeklyReport.setWeekInStudy(week.getWeekInStudy());
-            weeklyReport.setStartDate(week.getStartDate());
+        
+        StudyReportWeek week = report.getWeekReport();
+        weeklyReport.setSearchableLabels(week.getSearchableLabels());
+        weeklyReport.setRows(week.getRows());
+        weeklyReport.setByDayEntries(week.getByDayEntries());
```

Comment:
These setters were removed from StudyReportWeek.

Reply:
Those setters are there. weeklyReport is a WeeklyAdherenceReport, not a StudyReportWeek.